File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.py
def solution(s):
    return s.isdigit() and (len(s) == 4 or len(s) == 6)
# isdigit(): 문자열이 숫자로만 이루어져 있는지 확인


# 다른 풀이
return s.isdigit() and len(s) in [4,6]

# 다른 풀이
try:
    int(s)
except:
    return False
return len(s) == 4 or len(s) == 6

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        boolean answer = true;
        
        for(String str : s.split("")) {
            if(!(s.length() == 4 || s.length() == 6)){
                answer = false;
            }
            
            if(!Character.isDigit(str.charAt(0))) {
                answer = false;
            }
        }
        
        return  answer;
    }
}